### PR TITLE
Increase mimimum number of warmup runs to 2

### PR DESCRIPTION
**Pull request by t-vi**
The JIT will typically need two warmup runs to do profiling and optimization.
This is not the perfect solution but it will substantially reduce the number of surprised people when the docs say torch.utils.benchmark.Timer takes care of warmup.

### CODE REVIEW

**Review by robieta — APPROVED** [review]
TIL. Yeah, I agree it's a less than ideal heuristic but this is a better one. Thanks for the PR!
